### DISCUSSION TITLE
BT not displaying with Phidget/HUB

### DISCUSSION THREAD

**Discussion by wpschick** [artisan-roaster-scope/artisan#855] [Q&A]
I have installed the updated Artisan software and I am using 2 Thermocouples with my roaster. When I look at Phidget Console both TCs are outputting correct data (Mac Os). I.e. everything appears to be connected correctly with hardware and set-up. 

However, in Artisan only the ET is showing, the BT doesn't have an output.

I reinstalled Artisan (for mac) and configured using 'Machine' -- Phidget

What am I missing? Thanks in advance for your suggestion.

Warren

**Comment by bigl00z3** [comment]
Have you also updated your Phidget libraries to the [latest version](https://www.phidgets.com/docs/OS_-_macOS)?

Maybe specifiy which Phidget model you are using (e.g. TMP1101) and also upload your Artisan settings here. This will help other user in trying to help you.

Nico

**Reply by serave** [reply]
I am having a similar issue but, I am getting the other way around. I get BT but not ET. I am using the newer Phidget 1048_2B. 
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/1471836/161391939-a1046f24-a2d6-4529-870b-294e7258230a.png">

I got my BT on Channel 0 with a K type thermocouple and my ET on Channel 3 with a K type thermocouple. 
Using artisan on MAC OS Monterrey 12.2.1
Artisan 2.6.0 
<img width="978" alt="image" src="https://user-images.githubusercontent.com/1471836/161391014-9b88da46-4781-4ca7-a072-1a241ac930e1.png">
<img width="959" alt="image" src="https://user-images.githubusercontent.com/1471836/161391992-9cb02063-ebe5-4f9d-9866-b5a70755eda6.png">
<img width="964" alt="image" src="https://user-images.githubusercontent.com/1471836/161392005-c9f4b7c1-cd63-4a46-96b2-9b800bed87cf.png">
<img width="961" alt="image" src="https://user-images.githubusercontent.com/1471836/161392022-2f7ca137-a417-49b0-ae87-51efe2175b16.png">

**Reply by wpschick** [reply]
<img width="957" alt="Screen Shot 2022-04-03 at 7 31 28 PM" src="https://user-images.githubusercontent.com/102184753/161464526-6b2f6509-db20-4bfd-ae46-60b01f734cec.png">
<img width="937" alt="Screen Shot 2022-04-03 at 7 30 54 PM" src="https://user-images.githubusercontent.com/102184753/161464528-b560ab1e-0d01-4988-af84-1c9a33282443.png">

**Reply by wpschick** [reply]
I I have the VINT Hub Phidget HUB 0001_0 and the 4x Thermocouple Phidget TMP 1101_1
The BT probe is plugged into 4x Thermocouple Phidget  slot 2, and the ET probe is in slot 1

**Reply by wpschick** [reply]
Is this the same issue as described below where I need to plug BT into port 1 and ET in port 0?

**Comment by bigl00z3** [comment]
Dear @serave 

Looking at your config:
<img width="305" alt="image" src="https://user-images.githubusercontent.com/8886922/161400422-12fb27c6-1b93-48a2-8987-f538a5aa1fda.png">

You have selected port 0 & 1 of the 1048

- ET has to be on port 0
- BT has to be on port 1 (not 3)

If you want to connect (for any other non obvious reason) your BT to an other port on the 1048  (e.g. in your case "3" instead of "1"), you can, but you will need to add an E`xtra Device` and then map to your BT to this device under `Symb ET/BT `for example to `Y3` (`Y1` and `Y2` are already used by ET & BT as described in the help, extra devices are using other following symbolic terms, Y3, Y4, Y5...)

Nico

**Reply by serave** [reply]
Thank you so much Nico. I’ll try to connected as recommended. I kind of figure there default ports but I couldn’t find a reference on which ones to use.

thanks again 🙏🏽
